lib/browser/rpc-server.js

```diff
@@ -440,7 +440,8 @@ ipcMain.on('ELECTRON_BROWSER_WINDOW_CLOSE', function (event) {
   event.returnValue = null
 })
 
-ipcMain.on('ELECTRON_BROWSER_SANDBOX_LOAD', function (event, preloadPath) {
+ipcMain.on('ELECTRON_BROWSER_SANDBOX_LOAD', function (event) {
+  const preloadPath = event.sender._getPreloadPath()
```

Comment:
While we're in the neighborhood, the code below this

```js
   event.returnValue = {
     preloadSrc: preloadSrc,
     preloadError: preloadError,
     ...
```

could be updated to use ES6 enhanced object literals:

```js
   event.returnValue = {
     preloadSrc,
     preloadError,
     ...
```

Reply:
@ckerr done